### PR TITLE
Fix env file detection (make environment names uppercase)

### PR DESCRIPTION
**Pull request by 930C**
## Description

Environment Names should all be uppercase.

Fixes # (issue reference, if applicable)

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [x] Locally tested the according endpoints
- [ ] Ran E2E-Tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main.go [hunk 1]
```diff
@@ -22,14 +23,20 @@ func main() {
...
 	db_port := os.Getenv(fmt.Sprintf("%s_DB_PORT", environment))
 	db_user := os.Getenv(fmt.Sprintf("%s_DB_USER", environment))
 	db_password := os.Getenv(fmt.Sprintf("%s_DB_PASSWORD", environment))
 	db_name := os.Getenv(fmt.Sprintf("%s_DB_NAME", environment))
 
+	// Check if environment variables are set
+	if db_host == "" || db_port == "" || db_user == "" || db_password == "" || db_name == "" {
+		log.Fatalf("Required environment variables are not set")
+		os.Exit(1)
+	}
```
Comment (Frecherenkel60) [inline src/main.go:38]: Wollen wir dafür ne kleine Helper-Funktion schreiben? Sollte auch mit dem CodeScene-Problem helfen. Also im Utils-Package iwie `stringutils.go` und darin so eine Funktion:


```go
package utils

func containsEmptyString(strings ..string) bool {
    for _, s := range strings {
        if s == "" {
            return true
        }
    }
    
    return false
}
```

Und dann sowas wie:

```suggestion
        if containsEmptyString(db_host, db_port, db_user, db_password, db_name) {
		log.Fatalf("Required environment variables are not set")
		os.Exit(1)
	}
```

Comment (930C) [reply src/main.go:38]: Klingt gut. soll ich das Commit

Comment (930C) [reply src/main.go:38]: Klingt gut. Kannst du das Commit für mich ausführen, ich müsste jetzt los

**Review by Frecherenkel60 — APPROVED** [review]

**Review by Frecherenkel60 — APPROVED** [review]

**Comment by Frecherenkel60** [comment]
Weird, wieso hab ich 2x approved

**Pull request merged**